python-shell/src/gafferpy/gaffer_config.py

```diff
@@ -17,9 +17,10 @@
 """
 This module contains Python copies of Gaffer config java classes
 """
+import inspect
+import sys
```

Comment:
Why was it that these weren't here before?

Reply:
Before it called `from gafferpy.gaffer_core import *`, but as I cleaned up the imports to be explicit these need to be added.

Reply:
I suspected that, wasn't sufficiently familiar with Python to know it imported imports like that when `*` was used.